src/host/libpixyusb_swig/get_exposure.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+
+##
+# @file get_exposure.py
+# @brief This tool reads the Pixy's gain/exposure settings after certain program settings of the Pixy.
+#        The tool revealed a gain/exposure inconsistency in IR-Lock's MarkOne firmware. When used in-flight the
+#        MarkOne's gain/exposure is 0xFE and 0x01 respectively. When used with the computer application Pixymon, the
+#        gain/exposure changes to 0x00 and 0x20 respectively. The most likely reason for this inconsistency is because
+#        with the in-flight setting of 0x01FE, the displayed image on Pixymon has red/blue artifacts due to shifting of
+#        the bayer pattern. It is not clear why the camera sensor has this shift when the gain is high and exposure is
+#        low.
```

Comment:
Please forgive my ignorance.... what is the context in which this python script will execute?  We run it once after the Pixy is configured?  Or is it more often than that?

Reply:
I developed this script to prove that the IR Lock's MarkOne firmware is using two different gain/exposure settings. One setting when used with Pixymon (like a USB webcam) and one setting when used via I2C (in flight) with the Cube. 

No one should need to run this but if they want proof then they can run it.